keyboards/tweetydabird/lotus58/info.json

```diff
@@ -0,0 +1,154 @@
+{
+    "manufacturer": "Tweetys Wild Thinking",
+    "keyboard_name": "Lotus 58 Glow (QMK)",
+    "maintainer": "TweetyDaBird",
+    "bootloader_instructions": "Short marked pads on PCB, or hold top-outer key when plugging in each hand.",
+    "build": {
+        "lto": true
+    },
+   "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {
+                "pin_a": "F5",
+                "pin_b": "F4",
+                "resolution": 2
+            }
+        ]
+    },
+    "features": {
+        "audio": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "grave_esc": false,
+        "magic": false,
+        "mousekey": false,
+        "nkro": true,
+        "oled": true,
+        "rgblight": true,
+        "space_cadet": false,
+        "split": true,
+        "tap_dance": false,
+		"tri_layer": true
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B2", "B3", "B6", "F7", "F6"],
+        "rows": ["D4", "C6", "D7", "E6", "B4"]
+    },
+    "matrix_size": {
+        "cols": 6,
+        "rows": 10
+    },
+    "pin_compatible": "promicro",
+    "processor": "atmega32u4",
+    "rgblight": {
+        "brightness_steps": 17,
+        "hue_steps": 17,
+        "led_count": 70,
+        "pin": "D3",
+        "saturation_steps": 17,
+        "split": true,
+        "split_count": [35, 35]
+    },
+    "split": {
+        "enabled": true,
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {
+                        "pin_a": "F4",
+                        "pin_b": "F5",
+                        "resolution": 2
+                    }
+                ]
+            }
+        },
```

Comment:
```suggestion
        },
        "bootmagic": {
                "matrix": [5, 0]
        },
```

Reply:
I'm leaving these for now, since there are others in there that doesn't work in the .JSON

Reply:
This is missing the part for the main hand. Editing it manually.